service/src/main/java/com/myhome/services/springdatajpa/UserSDJpaService.java

```diff
@@ -59,7 +59,11 @@
     if (userRepository.findByEmail(request.getEmail()) == null) {
       generateUniqueUserId(request);
       encryptUserPassword(request);
-      return Optional.of(createUserInRepository(request));
+      User newUser = createUserInRepository(request);
+      SecurityToken emailConfirmToken = securityTokenService.createEmailConfirmToken(newUser);
+      //mailService.sendAccountCreated(newUser, emailConfirmToken);
```

Comment:
This commented because I worked simultaneously on #196, #211 and #213 so i can't fully separate changes on several pull requests